h/datasetjson.h

```diff
@@ -23,6 +21,9 @@
 
 #define SAF_AUTHORIZATION_READ 0x04
 #define SAF_AUTHORIZATION_UPDATE 0x08
+#define LOG_COMP_ID_MVD_SERVER 0x008F000300010000
+#define DATASET_PATH_MAX 44
+#define DATASET_MEMBER_MAXLEN 58
```

Comment:
How’d you get 58? A dataset member has a max length of 8.

44+8 = 52

Even if you include the (), I’m still off by 4.

Regardless, if you need a max length of something made up by two different values, just add them together.

For example,

#define DS_NAME_MAX_LEN 44
#define DS_MEMBER_NAME_MAX_LEN 8

char fullPath[DS_NAME_MAX_LEN + DS_MEMBER_NAME_MAX_LEN + 2 + 1] /* +2 for () and +1 for null

If anyone wanted to use that constant, it would just be wrong.

You could also do 

#define DS_NAME_PDS_MAX_LEN DS_NAME_MAX_LEN + DS_MEMBER_NAME_MAX_LEN + 2

Reply:
58 come from 44 + 8 + ( ) + //' + '